src/dialogs/more-info/ha-more-info-history.ts

```diff
@@ -134,19 +161,29 @@ export class MoreInfoHistory extends LitElement {
         return;
       }
     }
-    if (!isComponentLoaded(this.hass, "history")) {
+    if (!isComponentLoaded(this.hass, "history") || this._subscribed) {
       return;
     }
-    this._stateHistory = await getRecentWithCache(
+    this._subscribed = subscribeHistoryStatesTimeWindow(
```

Comment:
Should we check here if `this._subscribed` is undefined, and if not, unsub?

Reply:
Could always be a race. Good idea. Will adjust